test/general/backend_inclusion.pass.cpp

```diff
@@ -0,0 +1,69 @@
+// -*- C++ -*-
+//===-- backend_inclusion.pass.cpp ----------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+
+#include "support/utils.h"
+
+// The following test suites verify that the unused backend is disabled.

```

Comment:
I'd say "Verify that deactivated backends are not accessed, by checking respective header guard macros"